config/environments/uat.rb

```diff
@@ -82,4 +82,6 @@
   config.urs_client_id = services['urs'][Rails.env.to_s][services['earthdata'][config.echo_env]['urs_root']]
 
   config.cmr_client_id = ENV['cmr_client_id'] || 'edsc-prod'
+
+  config.opensearch_client_id = ENV['opensearch_client_id'] || 'edsc-prod'
```

Comment:
Can you just reuse the CMR client id instead of configuring them separately? I can't think of a case where we'd want them to be different.